git-hooks/post-checkout

```diff
@@ -1,3 +1,2 @@
-#!/usr/bin/env sh
-
-tar -C ./Carthage -xzf ./Carthage/Build.tar.gz
+#!/usr/bin/env bash
+exec git submodule update
```

Comment:
oh, also i didn't do `git submodule sync` and it seems to be working, but i'm not sure whether i need to - i'm a little unsure after looking at the manpage.